includes/blocks/media-aside/render.php

```diff
@@ -60,10 +60,10 @@ function benenson_render_media_aside_block( array $attributes = [] ) {
 				);
 			} else {
 				printf(
-					'<div><div class="inlineVideo mediaAside-videoContainer"><div class="inlineVideo-poster" style="background-image:url(%s);"></div><a class="btn"><i class="play-icon">%s</i></a><iframe width="560" height="349" data-src="%s" frameborder="0" allow="" allowfullscreen></iframe></div></div>',
+					'<div><div class="inlineVideo mediaAside-videoContainer"><div class="inlineVideo-poster" style="background-image:url(%s);"></div><a class="btn" data-inline-embed="%s"><i class="play-icon">%s</i></a></div></div>',
```

Comment:
I missed this previously, sorry, but `background-image:url(%s);` could use quotes, a la `background-image:url(\'%s\');` (escapes are for printf). I've known URIs that have broken the attribute value when quotes haven't been used.